brain-marks/Categories/Views/CategorySheetView.swift

```diff
@@ -11,7 +11,7 @@ struct CategorySheetView: View {
     
     @Binding var editCategory: AWSCategory?
     @Binding var categorySheetState: CategoryState
-    @ObservedObject var parentVM: CategoryListViewModel
+    @Binding var lastEditedCategoryID: String
```

Comment:
It wasn't necessary to pass the entire parent VM into CategorySheetView since it was only changing the lastEditedCategoryId. Instead passing this value as a Binding will accomplish the same result.